src/libs/ReportUtils.js

```diff
@@ -3152,6 +3152,7 @@ function shouldReportBeInOptionList(report, currentReportId, isInGSDMode, betas,
         !report ||
         !report.reportID ||
         !report.type ||
+        report.reportName === undefined ||
```

Comment:
Have you found case of reportName being set to null or empty string but need to be in options list?
```suggestion
        !report.reportName
```

Reply:
ok let's revert to `undefined` then